WindowsServerDocs/storage/storage-spaces/drive-symmetry-considerations.md

```diff
@@ -0,0 +1,175 @@
+| 2 x NVMe Model X (cache)    | 2 x NVMe Model X (cache)    | 2 x NVMe Model <mark>Z</mark> (cache) |
+| 10 x SSD Model Y (capacity) | 10 x SSD Model Y (capacity) | 10 x SSD Model Y (capacity)  |
+
+This is supported.
+
+### ![supported](media/drive-symmetry-considerations/supported.png) Supported: different models within server
+
+Every server uses some different mix of HDD models "Y" and "Z", which are very similar.
+
+| Server 1                       | Server 2                       | Server 3                       |
+|--------------------------------|--------------------------------|--------------------------------|
+| 2 x SSD Model X (cache)        | 2 x SSD Model X (cache)        | 2 x SSD Model X (cache)        |
+| <mark>7</mark> x HDD Model Y (capacity) | <mark>5</mark> x HDD Model Y (capacity) | <mark>1</mark> x HDD Model Y (capacity) |
+| <mark>3</mark> x HDD Model Z (capacity) | <mark>5</mark> x HDD Model Z (capacity) | <mark>9</mark> x HDD Model Z (capacity) |
+
+This is supported.
+
+### ![supported](media/drive-symmetry-considerations/supported.png) Supported: different sizes across servers
+
+The first two servers use 4 TB HDD but the third server uses very similar 6 TB HDD.
+
+| Server 1                | Server 2                | Server 3                    |
+|-------------------------|-------------------------|-----------------------------|
+| 2 x 800 GB NVMe (cache) | 2 x 800 GB NVMe (cache) | 2 x 800 GB NVMe (cache)     |
+| 4 x 4 TB HDD (capacity) | 4 x 4 TB HDD (capacity) | 4 x <mark>6</mark> TB HDD (capacity) |
+
+This is supported, although it will result in stranded capacity.
+
+### ![supported](media/drive-symmetry-considerations/supported.png) Supported: different sizes within server
+
+Every server uses some different mix of 1.2 TB and very similar 1.6 TB SSD.
+
+| Server 1                   | Server 2                   | Server 3                 |
+|----------------------------|----------------------------|--------------------------|
+| 3 x 1.2 TB SSD (cache)     | 2 x 1.2 TB SSD (cache)     | 4 x 1.2 TB SSD (cache)   |
+| 1 x <mark>1.6 TB</mark> SSD (cache) | 2 x <mark>1.6 TB</mark> SSD (cache) | -                        |
+| 20 x 4 TB HDD (capacity)   | 20 x 4 TB HDD (capacity)   | 20 x 4 TB HDD (capacity) |
+
+This is supported – note that every server has 4 total SSD.
+
+### ![unsupported](media/drive-symmetry-considerations/unsupported.png) Not supported: different types of drives across servers
+
+Server 1 has NVMe but the others don't.
+
+| Server 1             | Server 2            | Server 3            |
+|----------------------|---------------------|---------------------|
+| 6 x <mark>NVMe</mark> (cache) | -                   | -                   |
+| -                    | 6 x SSD (cache)     | 6 x SSD (cache)     |
+| 18 x HDD (capacity)  | 18 x HDD (capacity) | 18 x HDD (capacity) |
+
+This is not supported. The types of drives must be the same in every server.
```

Comment:
As mentioned before, it'd be good to do a search and replace to put contractions in for "is not" "that is", etc.